docs/SpatialEpigenome.rst

```diff
@@ -304,18 +289,16 @@ calling even if it has already been performed. This step is done to identify rep
 
 Add peak matrices
 ------------------------
-
-Use the addPeakMatrix function to call on proj_in_tissue to add a peak matrix to the object. This matrix is used to store the peak calls, which are 
-regions of the genome that show an enrichment of reads when compared to a background. This step is done to create a data structure that can efficiently 
-store and retrieve the peak calls, which can be used for downstream analysis.::
+To store and retrieve peak calls efficiently, use the addPeakMatrix function to add a peak matrix to the proj_in_tissue object. This matrix will store the peak calls which are regions of the genome that show an enrichment of reads when compared to a background.::
 
    proj_in_tissue <- addPeakMatrix(proj_in_tissue)
 
 
 Motif enrichment (Deviation)
 ----------------------------------------
 **Add motif annotations** 
-First, we check if motif annotations are already present in the project. If not, we add motif annotations to the ArchR project using the addMotifAnnotations function. If the data species is "hg38" or "mm10", we use the "cisbp" motif set. Otherwise, we use the "encode" motif set and obtain the species information from the project's genome.::
+
+Check if motif annotations are already present in the project. If not, use the addMotifAnnotations function to add them to the ArchR project. The motif set used will depend on the data species. If the species is "hg38" or "mm10", the "cisbp" motif set is used. Otherwise, the "encode" motif set is used and the species information is obtained from the project's genome.::
```

Comment:
You can get rid of the if-else code block and just keep the line of code to add motif annotations